deptrack-devops/src/Devops/Cli.hs

```diff
@@ -26,22 +26,21 @@ import qualified Data.ByteString.Base64.Lazy as B64
 import           Data.ByteString.Lazy        (ByteString)
 import           Data.Tree                   (Forest)
 import           Data.Typeable               (Typeable)
-import           DepTrack                    (GraphData, buildGraph,
-                                              evalDepForest1)
+import           DepTrack                    (GraphData, buildGraph, evalDepForest1)
 import           Prelude                     hiding (readFile)
 import           System.Environment          (getArgs, getExecutablePath)
 
-import           Devops.Actions              (concurrentTurndown,
-                                              concurrentTurnup, concurrentUpkeep, display, defaultDotify,
-                                              dotifyWithStatuses,
-                                              listUniqNodes, checkStatuses)
+import           Devops.Actions              (checkStatuses, concurrentTurndown, concurrentTurnup, concurrentUpkeep, defaultDotify,
+                                              display, dotifyWithStatuses, listUniqNodes, sequentialTurnDown, sequentialTurnup)
 import           Devops.Base                 (DevOp, OpUniqueId, PreOp, preOpUniqueId)
 
 --------------------------------------------------------------------
 
 data Method =
     TurnUp
+  | SequentialTurnUp
```

Comment:
I'd rather modify the TurnUp / TurnDown constructors to have a `data Synchronicity = Synchronous | Asynchronous` parameters. I'd rather the Method first level of constructor to represent orthogonal methods to operate on the graph rather than mechanisms (i.e., what we do vs. how we do).

I'm fine with the proposed parser/marshaller in `methodArg` and `appMethod` .